bin/tsds_adjust_metafields.pl

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/perl
+
```

Comment:
A comment up here explaining what this is for would be good. This isn't really a generic part of TSDS, it's more like a one time upgrade script that we'll use once and then throw away (or rather, just keep around for a bit as reference).